src/current/_includes/unsupported-version.md

```diff
@@ -105,7 +105,7 @@ Today date: {{ today | date: "%Y-%m-%d" }} <br />
 
 {% capture ga_maintenance_message %}
       {% if today <= m and today > a %}
-      {{site.data.alerts.callout_danger}}
+      {{site.data.alerts.callout_version}}
```

Comment:
This is the one that shows up purple. Nothing shows up green now, I explain it below the screenshot in the description.